DailyDemo.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -1,12 +1,12 @@
 {
   "pins" : [
     {
-      "identity" : "daily-client-ios",
+      "identity" : "daily-client-ios-internal",
```

Comment:
We'll want to update this before merging.

Reply:
updated in https://github.com/daily-demos/daily-ios-demo/commit/6b873310169e4b2bbe90618a843666f9b090dc1f